terraform/api/k8s/main.tf

```diff
@@ -3,6 +3,25 @@ resource "random_string" "password" {
   special = false
 }
 
+resource "kubernetes_resource_quota" "gcp-critical-pods" {
+  metadata {
+    name = "gcp-critical-pods"
+    namespace = var.namespace
+  }
+  spec {
+    hard = {
+      pods = "1G"
+    }
```

Comment:
Why it is `1G`?

Reply:
1G is the default hard limit on the Quota in the other namespace, I kept it identical in the convox system namespace also. I checked, it says 1G is 1000 pods limit.